test/utils/fizzy_engine.cpp

```diff
@@ -72,9 +73,54 @@ bytes_view FizzyEngine::get_memory() const
     return {m_instance->memory->data(), m_instance->memory->size()};
 }
 
-std::optional<WasmEngine::FuncRef> FizzyEngine::find_function(std::string_view name) const
+namespace
 {
-    return fizzy::find_exported_function(m_instance->module, name);
+FuncType translate_signature(std::string_view signature)
+{
+    std::string_view inputs(signature.begin(), signature.find(":"));
+    std::string_view outputs(inputs.end() + 1);  // Is this a terrible hack?
```

Comment:
```suggestion
    const auto delimiter_pos = signature.find(":");
    assert(delimiter_pos != std::string_view::npos);
    std::string_view inputs(signature.begin(), delimiter_pos);
    std::string_view outputs(delimiter_pos + 1, signature.end());
```

Reply:
```
/Users/alex/Projects/fizzy/test/utils/fizzy_engine.cpp:111:22: error: no matching constructor for initialization of 'std::string_view' (aka 'basic_string_view<char>')
    std::string_view outputs(delimiter_pos + 1, signature.end());
                     ^       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
@gumb0 any suggestions?

Reply:
`&signature[delimiter_pos] + 1`.

Reply:
But probably should be easier to do `signature.substr(0, delim)` and `signature.substr(delim+1)`.

Reply:
Yeah that's nicer, pushing that.